legacy-client.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
```

Comment:
Suggest putting this file inside `tests` to keep things tidy.

Reply:
Not sure where we are using this. I can remove it, but it is the same as `docker-compose.client.yml`.

Reply:
Yes, `version` must stay. I meant the file, not that line specifically.